workflow/tests/hpxml_translator_test.rb

```diff
@@ -642,9 +643,16 @@ def _verify_outputs(rundir, hpxml_path, results, hpxml)
       if hpxml.hvac_distributions.select { |d| d.air_type.to_s == HPXML::AirTypeFanCoil }.size > 0
         next if err_line.include? 'In calculating the design coil UA for Coil:Cooling:Water' # Warning for unused cooling coil for fan coil
       end
+      if hpxml.cooling_systems.select { |c| c.cooling_system_type == HPXML::HVACTypeRoomAirConditioner }.size > 0
+        next if err_line.include? 'AirLoopHVAC:UnitarySystem = ROOM AC UNITARY SYSTEM' # Warning for unused heating coil
+      end
```

Comment:
Why are we now including a heating coil for a room AC? Is it possible to exclude it?